msvc/build-env.bat

```diff
@@ -18,16 +18,18 @@ set OPENVPN_GIT=https://github.com/OpenVPN/openvpn.git
 set P7ZIP_URL=https://netcologne.dl.sourceforge.net/project/sevenzip/7-Zip/9.20/7za920.zip
 
 if "%ProgramFiles(x86)%"=="" set ProgramFiles(x86)=%ProgramFiles%
-if "%VSCOMNTOOLS%"=="" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Professional\Common7\Tools
-if not exist "%VSCOMNTOOLS%" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Community\Common7\Tools
-if not exist "%VSCOMNTOOLS%" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\Enterprise\Common7\Tools
+if "%VSCOMNTOOLS%"=="" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Professional\Common7\Tools
+if not exist "%VSCOMNTOOLS%" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Community\Common7\Tools
+if not exist "%VSCOMNTOOLS%" set VSCOMNTOOLS=%ProgramFiles(x86)%\Microsoft Visual Studio\2019\Enterprise\Common7\Tools
 if "%VSHOME%"=="" SET VSHOME=%VSCOMNTOOLS%\..\..
 if "%VCHOME%"=="" SET VCHOME=%VSHOME%\VC
-if not exist "%VSCOMNTOOLS%" SET VCHOME=%ProgramFiles(x86)%\Microsoft Visual Studio\2017\BuildTools\VC
+if not exist "%VSCOMNTOOLS%" SET VCHOME=%ProgramFiles(x86)%\Microsoft Visual Studio\2019\BuildTools\VC
 
 if "%P7Z%"=="" SET P7Z=tools\7za.exe
 if "%GIT%"=="" SET GIT=c:\Program Files\Git\bin\git.exe
 if "%NASM_DIR%"=="" SET NASM_DIR=c:\Program Files\NASM
 
 if "%TARGET%"=="" SET TARGET=%ROOT%\image
 if "%RELEASE%"=="" SET RELEASE=Release
+
+if "%RC%"=="" SET RC=rc
```

Comment:
Why this is needed?

Reply:
My computer got RC defined as "1" somehow by OpenSSL build system. If %RC% is set before invoking OpenSSL build, it leaves it.

Reply:
In fact I got the same problem on my environment, but wasn't able to trace origin. I solved it by running `set RC=`. Could you add a comment above that line with explanation (something like "on some environments OpenSSL defines RC as 1")?

Reply:
Sure. Fixed and force-pushed (to upgrade existing commit, rather than introducing new).